status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.174.8",
-    "commit-sha1": "8a3e71378f7208f75bd688c02b0ae5c43ca600f2",
-    "src-sha256": "10wn93xn6xnkg2d8slyygy9rfrwiargm49738bdjj1g4b81220bq"
+    "version": "v0.175.1",
```

Comment:
it's unclear to me why we're seeing Status-Go changes in this pr if not needed? 🤔

Reply:
@J-Son89 the status-go changes are needed. The `getRandomMnemonic` is from status-go. (although currently the feature is still hidden behind a feature-flag so either way is fine, but since the corresponding status-go pr is merged now so I am updating the version).

Reply:
okay I understand. I'm not entirely sure that's how we should be working though. 
As I understand it we don't want to merge Status-Go pr's until their corresponding mobile Pr's are tested and safe (this can be regression testing in the case of a feature flagged feature). Upgrading Status Go can still pull in other features from recent commits too 👍 
Also just so we're all on the same page this pr needs regression testing for the create account stuff because although the feature itself is feature flagged the code touching that flow is still altered here, so best to be safe 👌 

cc @pavloburykh

Reply:
@J-Son89 you are absolutely right. We cannot merge status go PRs until corresponding mobile PR is tested. If it has been done - it is probably better to revert those changes and wait for testing.

Reply:
Also, I am not sure we will be able to take this PR into testing until we cut mobile release branch. As in case of merge - there will be situation when mobile part of this PR will be included in the release while go part will not, because release go branch has been already cut.